Description: Use Debian nauty package instead of nauty shipped with M2.
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2016-04-11

--- a/M2/Macaulay2/packages/Nauty.m2
+++ b/M2/Macaulay2/packages/Nauty.m2
@@ -14,7 +14,7 @@
                  Email => "dwcook@eiu.edu",
                  HomePage => "http://ux1.eiu.edu/~dwcook/"}},
     Headline => "Interface to nauty",
-    Configuration => {"path" => ""},
+    Configuration => {"path" => "/usr/bin/nauty-"},
     PackageExports => {"EdgeIdeals"},
     DebuggingMode => false,
     Certification => {
--- a/M2/Macaulay2/packages/NautyGraphs.m2
+++ b/M2/Macaulay2/packages/NautyGraphs.m2
@@ -20,7 +20,7 @@
                  Email => "dcook8@nd.edu",
                  HomePage => "http://www.nd.edu/~dcook8"}},
     Headline => "Interface to nauty (Graphs fork)",
-    Configuration => {"path" => ""},
+    Configuration => {"path" => "/usr/bin/nauty-"},
     PackageExports => {"Graphs"},
     DebuggingMode => false,
 ), x -> x =!= null)
